src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractChangeNamespaceService.cs

```diff
@@ -589,11 +588,11 @@ private static async Task<ImmutableArray<ReferencedSymbol>> FindReferencesAsync(
             string newNamespace,
             CancellationToken cancellationToken)
         {
-            // Can't apply change to certain document, simply return unchanged.
+            // Can't apply change to certain document, returns null to indicate this.
             // e.g. Razor document (*.g.cs file, not *.cshtml)
```

Comment:
This comment doesn't make sense -- .g.cs files aren't razor files...?

Reply:
removed